util/src/main/java/com/redhat/lightblue/util/ServoGraphiteSetup.java

```diff
@@ -98,31 +110,89 @@ private static synchronized void doInitialize() {
             }
         }
 
-        String host = System.getenv("GRAPHITE_HOSTNAME");
-        LOGGER.debug("GRAPHITE_HOSTNAME environment variable in doInitialize() is " + host);
-        String port = System.getenv("GRAPHITE_PORT");
-        LOGGER.debug("GRAPHITE_PORT environment variable in doInitialize() is " + port);
         if (port == null) {
```

Comment:
I would suggest that it follows the same verification as the statd block, using Integer.valueOf and NumberFormatException